src/packages/ratmama/ratmama_parser.py

```diff
@@ -203,9 +203,21 @@ async def handle_ratmama_announcement(ctx: Context) -> None:
         platform=platform,
     )
     platform_signal = f"({rescue.platform.value.upper()}_SIGNAL)" if rescue.platform else ""
+
+    distance_str = "not found in the galaxy DB"
+    system = await asyncio.wait_for(ctx.bot.galaxy.find_system_by_name(system_name), timeout=2)
+    if system:
+        landmark_info = await asyncio.wait_for(ctx.bot.galaxy.find_nearest_landmark(system), timeout=2)
+        if landmark_info:
```

Comment:
[suggestion] Not sure in what context a system would, in fact, exist but no landmark data for it.
This case appears to be unhandled though, and the `find_nearest_landmark` *does* return an optional here soo....
we may want to explicitly handle that case